code/__DEFINES/_tick.dm

```diff
@@ -1,14 +1,19 @@
 /// Percentage of tick to leave for master controller to run
-#define MAPTICK_MC_MIN_RESERVE 70
+#define MAPTICK_MC_MIN_RESERVE 60
```

Comment:
try this at something much much lower like 40. tgmc doesn't need the mc to do as much.